specs/opcode/F1CALL_FASTATICCALL.md

```diff
@@ -72,7 +74,7 @@ callee_gas_left := min(gas_available - floor(gas_available / 64), gas)
 
 After switching call context, it does:
 
-1. Transfer `value`
+1. Transfer `value` (unnecessary for opcode `STATICCALL`)
```

Comment:
Can we assert `value` is zero for `staticcall` ?

Reply:
Added a constraint for `value == 0` if `STATICCALL`:
```
has_value = 1 - instruction.is_zero(value)
...
# Constrain value == 0 for opcode STATICCALL.
instruction.constrain_zero(has_value * (1 - is_call))
```

Reply:
better to change md doc : `1. Transfer `value` (zero  for opcode `STATICCALL`)` to make it clear.